idl/github.com/uber/cadence/shared.thrift

```diff
@@ -309,6 +309,7 @@ struct WorkflowExecutionInfo {
   80: optional WorkflowExecution parentExecution
   90: optional i64 (js.type = "Long") executionTime
   100: optional Memo memo
+  110: optional ResetPoints autoResetPoints
```

Comment:
Do you think it is possible to move this out to DescribeResponse? Reason is WorkflowExecutionInfo is used multiple places in visibility, unless you want such things in list result, I think it would be better to move out.

Reply:
Addressed offline.